cluster-scope/base/core/configmaps/thanos-ruler-custom-rules/configmap.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: thanos-ruler-custom-rules
+  namespace: open-cluster-management-observability
+data:
+  custom_rules.yaml: |
+    groups:
+      - name: openshift-storage
+        rules:
+        - alert: StoragePersistentVolumeFillingUp
```

Comment:
The problem with the `StoragePersistentVolumeFillingUp` alert is that we only care about that *for some volumes*. If you, a user, are running an application on the cluster and you are filling up your PV, we don't need an alert for that -- we only want alerts for things that will impact cluster services or applications that are explicitly maintained by NERC staff.

Can we limit this to alert only on volumes with specific labels (or perhaps only to PVCs in specific namespaces)?

Reply:
Sure @larsks I was able to limit the PVCs with an `expr` like this: 
https://github.com/OCP-on-NERC/nerc-ocp-config/pull/103/files#diff-f62ccb9a4fc420c5eb2adad7f01ffa3346f9c200cc6140dcca0354587f3ab299R16

```
expr: kubelet_volume_stats_available_bytes{persistentvolumeclaim=~"db-noobaa-db-pg-.*|metrics-backing-store-noobaa-pvc-.*|noobaa-default-backing-store-noobaa-pvc-.*"}/kubelet_volume_stats_capacity_bytes < 0.10
```
You can also add a `, namespace="..."` as well.

Reply:
@larsks does this resolve the CR?